### ISSUE TITLE
Fails for webpack 2.0.0-beta.25

### ISSUE THREAD

**Issue by isaacplmann** [rxaviers/skip-amd-webpack-plugin#1]
```
webpack: Using compiler.parser is deprecated.
Use compiler.plugin("compilation", function(compilation, data) {
  data.normalModuleFactory.plugin("parser", function(parser, options) { parser.plugin(/* ... */); });
}); instead.   It was called at SkipAMDPlugin.apply (project\path\node_modules\skip-amd-webpack-plugin\index.js:19:19).
```

Working on a PR.

**Comment by rxaviers** [comment]
Excellent! Thx

**Comment by isaacplmann** [comment]
I got stuck on the PR and found a workaround for now.  I'll circle back when I get a chance.

**Comment by necolas** [comment]
Any follow up? What was the workaround you used?

**Comment by esbenp** [comment]
This still fails for webpack `2.2.0-rc.4`. @rxaviers what is the status on reviewing #2 ?

**Comment by rxaviers** [comment]
Released `0.1.1` with a fix. Please, let me know if you run into any problems.
